scripts/package-scalingo.sh

```diff
@@ -31,8 +31,8 @@ fi
 # Copy the files in /dist, ready to be served.
 cp -r tchap-$version/* dist/
 # Also make a tar file. Useful for releases.
-tar chvzf tchap-$version-$today.tar.gz dist
+tar chvzf tchap-$version-$CONFIG-$today.tar.gz dist
```

Comment:
May produce unwanted results if $CONFIG is not defined

Reply:
we get "tchap-2.4.0-rc.3--20220309.tar.gz" for example, it doesn't break. Ca seems ok to me.
I could use "tchap-2.4.0-rc.3-unknown-config-20220309.tar.gz"... Don't know if it adds a lot of information...
So I'll stick to that.
Thanks!